server-tests/api.tests.js

```diff
@@ -0,0 +1,32 @@
+import { chai } from "meteor/practicalmeteor:chai";
+
+describe("chatter api methods", function () {
+  let user;
+  let room;
+  let meteorUser;
+
+  before(function() {
+    meteorUser = Meteor.users.findOne({_id: "id_of_user_one"});
+
+    const userId = Chatter.addUser(meteorUser._id, "admin");
+    user = Chatter.User.findOne({_id: userId});
```

Comment:
When using `findOne` by Id, you can just pass the id as parameter without the object.